api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/VersionedEntity.java

```diff
@@ -1,14 +1,12 @@
 package uk.gov.dhsc.htbhf.claimant.entity;
 
 import lombok.*;
-import lombok.experimental.SuperBuilder;
 import org.javers.core.metamodel.annotation.DiffIgnore;
 
 import javax.persistence.*;
 
 @Data
 @NoArgsConstructor
-@SuperBuilder
```

Comment:
Removing as it has no affect unless the child class uses a super builder. Which it's not due to no superbuilder support from the intellij plugin.